themes/default/content/docs/guides/organizing-projects-stacks/_index.md

```diff
@@ -106,6 +106,28 @@ its associated Pulumi stack. Read more about
 
 Stacks have associated metadata in the form of name/value tags. You can assign custom tags to stacks (when logged into the [Pulumi Service backend]({{< relref "/docs/intro/concepts/state" >}})) to customize how stacks are listed in the [Pulumi Service](https://app.pulumi.com). For example, if you have many projects with separate stacks for production, staging, and testing environments, it may be useful to group stacks by environment instead of by project. To do this, you could assign a custom `environment` tag to each stack, assigning a value of `production` to each production stack, `staging` to each staging stack, etc. Then in the Pulumi Service, you'll be able to group stacks by `Tag: environment`. For more information, see [Stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}).
 
-## Examples
+## Example
+
+In practice, it's common to have a combination of approaches. This is an example of an orgaizational setup that leverages several different approaches to provide the most functionality and flexibility possible.
```

Comment:
```suggestion
In practice, it's common to have a combination of approaches. Let's build an example of an organizational setup that leverages several different approaches to provide the most functionality and flexibility possible.
```

Reply:
The text reads like there's example code to go in here. Was that intentional?

Reply:
i've been debating that. this page is already really long, so I was leery about adding example code, especially since it could be different depending on hyperscalers, etc.